Documentation/operations/upgrade.rst

```diff
@@ -283,27 +283,42 @@ combination is not listed in the table below, then it may not be safe. In that
-+=======================+=======================+=========================+===========================+
-| ``1.12.x``            | ``1.13.y``            | Minimal to None         | Clients must reconnect[1] |
-+-----------------------+-----------------------+-------------------------+---------------------------+
-| ``1.11.x``            | ``1.12.y``            | Minimal to None         | Clients must reconnect[1] |
-+-----------------------+-----------------------+-------------------------+---------------------------+
-| ``1.10.x``            | ``1.11.y``            | Minimal to None         | Clients must reconnect[1] |
-+-----------------------+-----------------------+-------------------------+---------------------------+
++-----------------------+-----------------------+----------------------------+---------------------------+
+| Current version       | Target version        | L3/L4 impact               | L7 impact                 |
++=======================+=======================+============================+===========================+
+| ``1.13.4``            | ``1.13.y``            | Lingering Connections[2]   | Clients must reconnect[1] |
++-----------------------+-----------------------+----------------------------+---------------------------+
+| ``1.13.x``            | ``1.13.4``            | Interrupted Connections[2] | Clients must reconnect[1] |
++-----------------------+-----------------------+----------------------------+---------------------------+
+| ``1.12.x``            | ``1.13.y``            | Minimal to None            | Clients must reconnect[1] |
++-----------------------+-----------------------+----------------------------+---------------------------+
+| ``1.11.x``            | ``1.12.y``            | Minimal to None            | Clients must reconnect[1] |
++-----------------------+-----------------------+----------------------------+---------------------------+
+| ``1.10.x``            | ``1.11.y``            | Minimal to None            | Clients must reconnect[1] |
++-----------------------+-----------------------+----------------------------+---------------------------+
 
 Annotations:
 
 #. **Clients must reconnect**: Any traffic flowing via a proxy (for example,
    because an L7 policy is in place) will be disrupted during upgrade.
    Endpoints communicating via the proxy must reconnect to re-establish
    connections.
+#. **Interrupted connections**: Existing connections may be interrupted.
+#. **Lingering connections**: Existing connections may linger for too long in
+   the conntrack table.
 
 .. _current_release_required_changes:
 
 .. _1.13_upgrade_notes:
 
+1.13.4 (to and from) Upgrade Notes
+----------------------------------
+
+* Existing connections may be interrupted when upgrading to v1.13.4, while they
+  may linger for too long when upgrading from v1.13.4.  This is due to a
+  difference in how the age of the connections is calculated when using
+  bpf-clock-probe (:gh-issue:`26955`).
+
```

Comment:
We should have a place documenting the helm flag change `bpf.clockProbe` -> `bpfClockProbe`

Reply:
The previous flag never worked. If we are writing something, it would be something like:
"the previous Helm flag `bpf.clockProbe` used in `values.yaml` didn't work, the correct flag name is `bpfClockProbe`"

I'm not against documenting it, but I don't know where I should put something like that.

Reply:
Usually we would document such changes here: https://docs.cilium.io/en/v1.13/operations/upgrade/#helm-options

Some phrasing similar to this could work:

> The Helm option `bpf.clockProbe` was not functional previously. The setting `bpfClockProbe` now configures the clock probe mechanism for BPF timestamps.

Reply:
Never mind my suggestion, it looks like you already addressed this. LGTM.